sql/parser/parse_test.go

```diff
@@ -614,6 +614,13 @@ CREATE TABLE a (b INT DEFAULT (SELECT 1))
                                         ^
 `,
 		},
+		{
+			`SELECT POSITION('high', 'a')`,
```

Comment:
I had a well-formed test case in `TestParse2`, but the test was failing because it would convert `POSITION('ig' in 'high')` to `POSITION('high', 'ig')`, which is not a valid command (see the other error case). Should I add the case to `TestParseSyntax`, or is there a better place that it can go?